Troubleshooting docs for ACTION_RESTORE_FROM_SNAPSHOT (#87692)
elastic/elasticsearch: docs/reference/troubleshooting/data/restore-from-snapshot.asciidoc

[[restore-from-snapshot]]
== Restore from snapshot

Elasticsearch is using snapshots to store a copy of your data outside a cluster. You can restore a snapshot to recover

> indices and data streams for which there are no copies of the shards in the cluster. This can happen after a deletion or

Reviewer: ```suggestion
indices and data streams for which there are no copies of the shards in the cluster. This can happen if the data (indices or data streams) was deleted or
```

> hardware failure.

Reviewer: ```suggestion
if the cluster membership changed and the current nodes in the system do not contain a copy of the data anymore.
```

> IMPORTANT: Restoring the missing data requires you to have a usable backup of the affected indices and data streams. Please do not

Reviewer: What does usable mean? Should we clarify that or remove "usable" ?

> IMPORTANT: Restoring the missing data requires you to have a usable backup of the affected indices and data streams. Please do not

Reviewer: ```suggestion
IMPORTANT: Restoring the missing data requires you to have a backup of the affected indices and data streams. Please do not
```

Author: Replaced the word `usable` in a way that is consistent with the rest of the document.
